docker/Dockerfile.tcf-dev

```diff
@@ -111,17 +142,15 @@ RUN SGX_SDK_FILE=sgx_linux_x64_sdk_2.7.101.3.bin \
 
 
 WORKDIR /tmp
+ENV OPENSSL_VER=1.1.1d
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER /tmp/openssl-$OPENSSL_VER/
+COPY --from=openssl_image /tmp/openssl-$OPENSSL_VER.tar.gz /tmp/
```

Comment:
We don't need to copy entire openSSL build artifacts here. copy the required openSSL build artifacts like config files, binaries and libraries ?

Reply:
I looked at it but it creates various symlinks etc which is difficult to manage. So, its better to copy the entire openSSL build.

Reply:
openssl is a 3rd party tool. Currently avalon is using 1.1.1d version of it. I see that artifacts generated have version appended to them. So, we need to selectively copy the artifacts with some number as hardcoding. For eg: it generates "libcrypto.so.1.1". In future, if we decided to move to some other version of openssl, then there will be many places that we need to make change according to new version for the artifacts. So, I would prefer not to fall into that. Lets keep it like that.

Reply:
We can copy *.so or selectively copy required directories from openssl build artifacts. We can change configure command to pass build output directory. From openssl build ouput ,copy only required directories needed in run time like bin, lib etc. Please investigate further. Refer https://www.howtoforge.com/tutorial/how-to-install-openssl-from-source-on-linux/ and/or https://wiki.openssl.org/index.php/Compilation_and_Installation

Reply:
I tried copying the required directories, files and config files etc. There doesn't seem to be any improvement in docker image size.